internal/commands/profile/init.go

```diff
@@ -103,6 +169,62 @@ func (i *InitOpts) configureOrgAndProject() error {
 	return nil
 }
 
+func (i *InitOpts) configureVaultSecrets() error {
+	// Retrieve apps associated with org and project ID
+	listAppReq := preview_secret_service.NewListAppsParamsWithContext(i.Ctx)
+	listAppReq.OrganizationID = i.Profile.OrganizationID
+	listAppReq.ProjectID = i.Profile.ProjectID
+	listAppResp, err := i.SecretService.ListApps(listAppReq, nil)
+	if err != nil {
+		return err
+	}
+
+	appCount := len(listAppResp.Payload.Apps)
+
+	if appCount <= 0 {
+		return fmt.Errorf("there are no valid vault secrets apps for your principal. Create one by visiting the HCP Portal (https://portal.cloud.hashicorp.com)")
```

Comment:
Should this be an error or just an info message printed to `Stderr`? Also should we instruct the user to use `hcp vault-secrets apps create` instead?

Reply:
good idea,  I was following a similar setup to proj, but makes sense to impart more info here.